### ISSUE TITLE
Modal "close" button has issues

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#645]
`<button type="button" class="ajax-modal-close close" data-dismiss="modal" aria-hidden="true">×</button>`

Not sure why this has `aria-hidden="true"` - and oddly, it's not hidden, just doesn't make sense. Current reads as "times button".

Better would be something like this:
`<button><svg aria-hidden="true"></svg><span class="sr-only">close</span></button>`
where the `<svg>` represents the "x", so that:

- sighted user sees X
- screen reader doesn't perceive the X but hears "close"
- button itself is not suppressed

BUT when the content is displayed directly and not in a modal, like so:
https://earthworks.stanford.edu/catalog/stanford-hs643sf0606/metadata

the button is either suppressed or disabled.

**Comment by mejackreed** [comment]
@jvine I went with a slightly alternative approach. I used the same markup for the close buttons that we do in upstream Blacklight giving us consistency around the behavior. I think this resolves the aria/screen reader issues mentioned as well. This is also the recommended BS4 pattern.

```erb
<button type="button" class="blacklight-modal-close close" data-dismiss="modal" aria-label="<%= t('blacklight.modal.close') %>">
  <span aria-hidden="true">&times;</span>
</button>
```

https://getbootstrap.com/docs/4.5/components/modal/#modal-components

**Comment by jvine** [comment]
Sure, that's fine.

Still leaves the question of a non-functional button in the taborder when the modal is opened as a page, but I think the a11y crawler encounters that more than a human would.
